daemon/execdriver/lxc/driver.go

```diff
@@ -85,16 +85,21 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		dataPath = d.containerDir(c.ID)
 	)
 
+	container, err := d.createContainer(c)
```

Comment:
Yes, `ProcessConfig` is being set up for the execution of the process (100 lines further down), and shouldn't affect the `createContainer` flow; I guess the important aspect of this PR is erroring out before allocating a tty (or basic console pipes) for the `ProcessConfig.Terminal` setting..